synapse/media/_base.py

```diff
@@ -151,7 +153,16 @@ def _quote(x: str) -> str:
     else:
         content_type = media_type
 
+    # Only hotserve "safe" mimetypes, force download everything else
+    disposition_type = "attachment"
+    for mime in HOTSERVE_CONTENT_TYPES:
+        if media_type.lower().startswith(mime):
+            disposition_type = "inline"
+            break
```

Comment:
From chatting internally it seems safer to just always use an `attachment` content disposition. There isn't really any benefit to using `inline` as things like `<img>` tags ignore the `Content-Disposition` header anyway.

Reply:
If `<img>` and `<video>` ignore inline that seems sensible